README.md

```diff
@@ -29,12 +29,12 @@ You can control who has access to a Sync Server by editing its `allowed-users.tx
 
 - [Install docker](https://docs.docker.com/get-docker/)
 - clone code
-- Build the container: `docker compose build map-sync`
+- Build the container: `docker-compose build map-sync`
```

Comment:
`docker compose` is preferred over `docker-compose` because the latter runs a separate program and may be outdated, while the former uses the builtin compose